### PR TITLE
Update readme API_URL env var and added warning 

### PR DESCRIPTION
**Pull request by cptechiegal**
Issue#4753: https://github.com/18F/FEC/issues/4753
Update readme API_URL env var and added warning about SQLA_CONN env var impact on DB updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,12 +117,17 @@ To load statutes into elasticsearch, run:
...
+#### Connecting to a RDS DB instance instead of local DB
+
 *Note: FEC and 18F members can set the SQL connection to one of the RDS boxes with:*
 
 ```
 export SQLA_CONN=<psql:address-to-box>
 ```
 
+Warning: never perform 'update all' when pointing to an RDS box via the SQLA_CON
+N env var
```
Comment (ccostino) [inline README.md]: It looks like either an errant press of the enter key made it in or the line wrap settings in your editor are causing things to actually kick down to the next line with a line break, which also causes extra whitespace to show up, e.g.,:

screen shot 2017-12-05 at 9 34 43 am

Please double check or just remove the extra space so we don't confuse folks who are trying to set things up and set this environment variable. :-)

File: README.md [hunk 2]
```diff
@@ -145,7 +150,7 @@ export SQLA_FOLLOWERS=<psql:address-to-replica-box-1>[,<psql:address-to-replica-
 2. Run:
 
    ```
-   export FEC_WEB_API_URL=http://localhost:5000
+   export FEC_API_URL=http://localhost:5000
```
Comment (ccostino) [inline README.md]: Nice catch, thank you!

**Comment by xtine** [comment]
To link to an issue correctly, I would also include the entire url of the issue into the PR description.

**Comment by cptechiegal** [comment]
Thanks Carlo and Christine for the review!

**Pull request merged**